config.yaml.example

```diff
@@ -12,3 +12,7 @@ customImage:
       limits:
         cpu: 1000m
         memory: 500Mi
+
+jobSubmission:
+  graphqlEndpoint: https://hub.celu.dev.primehub.io/graphql
+  graphqlSecret: b3f0c00afb6b8e37e3f9ea05c1695ece6f8758411950ed711a8f88687fb22c84
```

Comment:
Do we need to change to sth more general rather than real one?

Reply:
Done.